lib/SaferpayJson/Request/Transaction/InitializeRequest.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ticketpark\SaferpayJson\Request\Transaction;
+
+use JMS\Serializer\Annotation\SerializedName;
+use Ticketpark\SaferpayJson\Request\Container\Authentication;
+use Ticketpark\SaferpayJson\Request\Container\CardForm;
+use Ticketpark\SaferpayJson\Request\Container\Order;
+use Ticketpark\SaferpayJson\Request\Container\Payer;
+use Ticketpark\SaferpayJson\Request\Container\Payment;
+use Ticketpark\SaferpayJson\Request\Container\PaymentMeans;
+use Ticketpark\SaferpayJson\Request\Container\ReturnUrls;
+use Ticketpark\SaferpayJson\Request\Container\RiskFactors;
+use Ticketpark\SaferpayJson\Request\Container\Styling;
+use Ticketpark\SaferpayJson\Request\Container\Wallet;
+use Ticketpark\SaferpayJson\Request\Request;
+use Ticketpark\SaferpayJson\Request\RequestCommonsTrait;
+use Ticketpark\SaferpayJson\Request\RequestConfig;
+use Ticketpark\SaferpayJson\Response\Transaction\InitializeResponse;
+
+final class InitializeRequest extends Request
+{
+    use RequestCommonsTrait;
+
+    public const API_PATH = '/Payment/v1/Transaction/Initialize';
+    public const RESPONSE_CLASS = InitializeResponse::class;
+
+    public const PAYMENT_METHOD_AMEX = "AMEX";
+    public const PAYMENT_METHOD_BANCONTACT = "BANCONTACT";
+    public const PAYMENT_METHOD_BONUS = "BONUS";
+    public const PAYMENT_METHOD_DINERS = "DINERS";
+    public const PAYMENT_METHOD_DIRECTDEBIT = "DIRECTDEBIT";
+    public const PAYMENT_METHOD_JCB = "JCB";
+    public const PAYMENT_METHOD_MAESTRO = "MAESTRO";
+    public const PAYMENT_METHOD_MASTERCARD = "MASTERCARD";
+    public const PAYMENT_METHOD_MYONE = "MYONE";
+    public const PAYMENT_METHOD_UNIONPAY = "UNIONPAY";
+    public const PAYMENT_METHOD_VISA = "VISA";
+    public const PAYMENT_METHOD_VPAY = "VPAY";
```

Comment:
Hat es einen Grund, warum die payment Konstanten hier nochmal definiert werden und nicht die aus der wallet vverwendet werden?

Reply:
Sure, we can switch to English. Thanks for the explanation, if the "api" isn't really fixed here, then this is probably the safest way. From my point of view, that's done.